### DISCUSSION TITLE
Is there a way for a custom service to identify skipped tests?

### DISCUSSION THREAD

**Discussion by ajeetd** [webdriverio/webdriverio#11494] [Q&A]
Specifically does a custom service (https://webdriver.io/docs/customservices/) have a way of receiving information about test cases that are skipped?

Hooks (https://webdriver.io/docs/configurationfile) are not called for skipped test cases.

I am wondering whether I've missed something or whether this functionality does not currently exist.

**Comment by christian-bromann** [comment]
Custom services get information about tests in various hooks, e.g. `beforeTest` and `afterTest`. Which information are you looking for?

**Reply by ajeetd** [reply]
The beforeTest and afterTest hooks are not called for skipped tests, looking for a way to know what the skipped tests were.

**Reply by christian-bromann** [reply]
you might need a reporter for that which communicates with the custom service.
